lib/openstudio/workflow/util/model.rb

```diff
@@ -101,8 +101,33 @@ def translate_to_energyplus(model, logger = nil)
           # ensure objects exist for reporting purposes
           model.getFacility
           model.getBuilding
-          forward_translator = OpenStudio::EnergyPlus::ForwardTranslator.new
-          model_idf = forward_translator.translateModel(model)
+          ft = OpenStudio::EnergyPlus::ForwardTranslator.new
+
+          ft_options = @options[:ft_options]
+          if !ft_options.empty?
+            mapping = {
+              :runcontrolspecialdays => :setKeepRunControlSpecialDays,
+              :ip_tabular_output => :setIPTabularOutput,
+              :no_lifecyclecosts => :setExcludeLCCObjects,
+              :no_sqlite_output => :setExcludeSQliteOutputReport,
+              :no_html_output => :setExcludeHTMLOutputReport,
+              :no_variable_dictionary => :setExcludeVariableDictionary,
+              :no_space_translation => :setExcludeSpaceTranslation,
+            }
+
+            msg = "Custom ForwardTranslator options passed:\n"
+            mapping.each do |opt_flag_name, ft_method|
+              opt_flag = ft_options[opt_flag_name]
+              if !opt_flag.nil?
+                ft.method(ft_method).call(opt_flag)
+                msg += "* :#{opt_flag_name}=#{opt_flag} => ft.#{ft_method}(#{opt_flag})\n"
+              end
+            end
+
+            logger.info msg
+          end
+
+          model_idf = ft.translateModel(model)
```

Comment:
Pass options to ForwardTranslator before translating the model.

I suppose to some extent these options (and the `epjson` one recently added) should probably end up in the OSW schema as well so you could set it there instead

Reply:
Another question is how can we make this more resilient and avoid having to make sure openstudio and the openstudio-workflow-gem are updated in tandem? (perhaps we don't care enough to engineer a resilient system and are ok relying on manual labor here).

One thing could be that OpenStudio itself would have the `mapping` above, so we won't have to update 3 locations (the FT, the CLI args, and the workflow-gem). 

* Either the CLI passes it somehow, or perhaps,
* a `static std::pair<std::string, std::string> openstudio::energyplus::ForwardTranslator::optionMapping()` that could both be used in the CLI and on the workflow-gem to ensure that anything passed as CLI arg /retrieved on workflow-gem is actually supported for our given version.

Reply:
Yes on supporting OSW.  As discussed on slack any options set and passed in via CLI, those would override any option set in OSW.  Perhaps another check can go in `input/local.rb` and if the user_options are null then check the OSW file itself and return those.

Reply:
It's a good thought on how properly align CLI, FT and workflow-gem.  The bigger question is if the workflow-gem itself will be converted to C++ to support Python measures.